packages/@aws-cdk/aws-stepfunctions-tasks/lib/codebuild/start-build.ts

```diff
@@ -12,12 +12,187 @@ export interface CodeBuildStartBuildProps extends sfn.TaskStateBaseProps {
    * CodeBuild project to start
    */
   readonly project: codebuild.IProject;
+  /**
+   * Build output artifact settings that override, for this build only.
+   *
+   * @default - the latest ones already defined in the build project.
+   */
+  readonly artifactsOverride?: codebuild.IArtifacts;
+  /**
+   * A buildspec file declaration that overrides, for this build only.
+   *
+   * @default - the latest one already defined in the build project.
+   */
+  readonly buildspecOverride?: string;
```

Comment:
why a string? why not `codebuild.BuildSpec` ? 
is that because it might also need to be retrieved from state machine input?